### ISSUE TITLE
Flaky test: SSL Handshake WRAP produced 0 bytes.

### ISSUE THREAD

**Issue by rossabaker** [http4s/http4s#2192]
```
02:19:49.930 [InnocuousThread-4] WARN  o.h.blaze.pipeline.stages.SSLStage - SSL Handshake WRAP produced 0 bytes.
Status = OK HandshakeStatus = NEED_WRAP
bytesConsumed = 0 bytesProduced = 0
02:19:49.930 [InnocuousThread-4] WARN  o.h.blaze.pipeline.stages.SSLStage - SSL Handshake WRAP produced 0 bytes.
Status = OK HandshakeStatus = NEED_WRAP
bytesConsumed = 0 bytesProduced = 0
02:19:49.931 [InnocuousThread-4] WARN  o.h.blaze.pipeline.stages.SSLStage - SSL Handshake WRAP produced 0 bytes.
Status = OK HandshakeStatus = NEED_WRAP
bytesConsumed = 0 bytesProduced = 0
02:19:49.931 [InnocuousThread-4] WARN  o.h.blaze.pipeline.stages.SSLStage - SSL Handshake WRAP produced 0 bytes.
```

This happened on an [openjdk11 build](https://travis-ci.org/http4s/http4s/jobs/443490411#L3637), which makes me fear that http4s/blaze#233 is sticking our heads in the sand.

**Comment by rossabaker** [comment]
Again: https://travis-ci.org/http4s/http4s/jobs/444236484

**Comment by rossabaker** [comment]
I expect this is fixed by #2204.  Will reopen if it's not.

**Comment by rossabaker** [comment]
[:fire:](https://travis-ci.org/http4s/http4s/jobs/447372206#L3562)

**Comment by rossabaker** [comment]
https://travis-ci.org/http4s/http4s/jobs/447372379#L3243

**Comment by ghostdogpr** [comment]
We just got hit by that in production, right after we switched to JDK11 😢

**Comment by coltfred** [comment]
We also started seeing this after upgrading to jdk 11.

**Comment by sderosiaux** [comment]
Same here. It happens after several minutes, client idle.
AdoptOpenJDK (build 11.0.4+11) / http4s v0.21.0-M5

**Comment by rossabaker** [comment]
We're working on releases of 0.20.x and 0.21.x, but in the meantime, this should be fixed by explicitly upgrading to blaze-0.14.11.
